easypy/signals.py

```diff
@@ -335,6 +389,47 @@ def run_handler(handler):
         self.remove_handlers_if_exist(handlers_to_remove)
 
 
+@parametrizeable_decorator
+def signal(func, asynchronous=None, swallow_exceptions=False, log=True):
```

Comment:
Why `asynchronous=None` and not `asynchronous=False`?

Reply:
`asynchronous=False` means you are not **allowed** to put asynchronous handlers on the signal. `asynchronous=None` means both sync and async handlers are OK.

Reply:
> `asynchronous=False` means you are not **allowed** to put asynchronous handlers on the signal. `asynchronous=None` means both sync and async handlers are OK.

It is very unclear. Is it documented somewhere?

Reply:
I'm just replicating the behavior of the `Signal` class...